ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__arm64-stable.yaml

```diff
@@ -259,6 +259,46 @@ tests:
     test:
     - chain: cucushift-installer-check-cluster-health
     workflow: baremetal-lab-agent-install-pxe
+- as: gcp-ipi-compact-filestore-csi-f14
+  cron: 43 19 4,24 * *
+  steps:
+    cluster_profile: gcp-qe
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:arm64-latest
+    dependency_overrides:
+      OO_INDEX: quay.io/openshift-qe-optional-operators/aosqe-index:v4.15
+    env:
+      COMPUTE_NODE_TYPE: t2a-standard-4
```

Comment:
For compact/sno cluster, need a bigger size of master nodes, but does not need compute nodes.

Reply:
removed compute. 
now ipi-conf-gcp can't cover compact for arm after some pr merge, see [here](https://github.com/openshift/release/blob/master/ci-operator/step-registry/ipi/conf/gcp/ipi-conf-gcp-commands.sh#L24-L39) I will fix it in another pr.

Reply:
fix pr is here: https://github.com/openshift/release/pull/45543